tests/integration-tests/tests/common/scaling_common.py

```diff
@@ -16,12 +18,113 @@
 from remote_command_executor import RemoteCommandExecutor
 from retrying import RetryError, retry
 from time_utils import seconds
-from utils import get_compute_nodes_count
+from utils import CWMetric, get_compute_nodes_instance_count, publish_metrics_to_cloudwatch
 
 SCALING_COMMON_DATADIR = pathlib.Path(__file__).parent / "scaling"
 
 
-def get_compute_nodes_allocation(scheduler_commands, region, stack_name, max_monitoring_time):
+def scaling_target_condition(
```

Comment:
It would be good to have a description of this function

Reply:
added